src/java.smartcardio/share/native/libj2pcsc/pcsc.c

```diff
@@ -45,6 +45,7 @@
 #define dprintf2(s, p1, p2) printf(s, p1, p2)
 #define dprintf3(s, p1, p2, p3) printf(s, p1, p2, p3)
 #else
+#undef dprintf // GNU, POSIX.1-2008
```

Comment:
Shouldn't this be before the `#ifdef J2PSC_DEBUG`?  Otherwise, we'll get the same problem when
debugging is enabled.

Reply:
A more sweeping change would be to rename this dprintf macro to (say) debug_printf, make it
variadic, and get rid of the various dprintfN variants.  I'm okay with the current approach as an
unbreak-the-build fix though.

Reply:
Thanks for the advice. dprintf has been renamed to debug_printf and make it variadic.